### ISSUE TITLE
Static Pages Content Request from BAH MBS Self-Service Team

### ISSUE THREAD

**Issue by ccruzgra** [department-of-veterans-affairs/va.gov-team#61517]
## About your team

- Team name: MBS Self-Service
-	OCTO-DE product owner: Matt Self
-	Product manager: Catherine Hughes
-	Designer: Cindy Cruz
-	FE engineer: Kenny Santiago
- Accessibility specialist: None
- Content writer: None, usually designers
-	Product/team Slack channel: #va-gov-mbs
 
### What does your team need support for? Check all that apply.

- [x] Making revisions to one or more existing VA.gov pages
 
### Give a brief description of what your team needs support with:

My team needs support with making updates to the Pre-Need static page and the After You Apply pages, based on usability testing input from users. 
[Pre-Need Information/static page](https://staging.va.gov/burials-memorials/pre-need-eligibility/) 
[After You Apply page](https://staging.va.gov/burials-memorials/pre-need-eligibility/after-you-apply/)

For the Pre-Need static page:
- The content is quite lengthy overall on mobile devices
- It can be challenging for users to finally locate the green "Apply for pre-need eligibility 
- The note at the bottom is outdated and should be removed per client request. It states: "Note: We’ve received a large number of pre-need eligibility applications, and it’s slowing down our approval process. We’ll let you know when we’ve received your application and how long we think it’ll take to make a decision. We’re sorry for the delay."
- We need to ensure that its content aligns to the content communicated to users on form Introduction/landing page and Supporting Documents step in the form, as well as the paper form
- There is a lack of context for what "time of need" means as it relates to "pre-need" in the messaging at the top
- There is no guidance provided regarding Corey Shea Act applicants
- There is guidance under the By Mail copy for **How do I apply?** section that mentions "If that person is under 18 years" guidance that the client wishes to have removed as children under 18 with eligible Veteran sponsors, their eligibility would be determined at time of need and pre-need is not intended for them.

For the After You Apply page:
- The note at the top is outdated and should be removed per client request. It states: "Note: We’ve received a large number of pre-need eligibility applications, and it’s slowing down our approval process. We’ll let you know when we’ve received your application and how long we think it’ll take to make a decision. We’re sorry for the delay."
- The link to "Check your VA claim status" is misleading as it was determined in a separate issue that the Pre-Need submissions aren't being tracked in the Claims Status Tool, so this link needs to be removed. [Slack thread](https://dsva.slack.com/archives/CBU0KDSB1/p1689693560979439)
- Need a general content review on it
 
### Will this new product be released incrementally (for instance 25% of users initially)? 
Yes, we will be releasing incremental enhancements, but they will be seen by 100% of users as they are released.
  - [x] Yes
  - [ ] No
 
**Note:** If you checked yes, we'll reach out to discuss details about the content in the react widget.
  
### When do you expect to launch your product to 100% of users? 
Pre-Integration enhancements are expected to be completed by June 2024.
  
*Please provide an estimated date so our team can create other relevant tickets.*

### When does this work need to be completed?
- Estimated launch date: Dec. 2023
- Estimated staging review date: Nov. 2023
- Content, Accessibility, and IA work needed by: Aug./Sept. 2023

### Will this work be going through the Collaboration Cycle?
- [x] Yes 
- [ ] No

### If no ...
- Skip to the _supporting artifacts_ section, below.
 
#### Please provide the link to your Collaboration Cycle ticket: 
- Ticket (number): [#54528](https://github.com/department-of-veterans-affairs/va.gov-team/issues/54528) 
- Collab. Cycle Ticket Opened (date): 3/2/23
 
#### Please check all of the phases in the Collaboration Cycle you have completed: 
- [x] Design Intent
- [ ] Midpoint Review
- [ ] Staging
 
##### Design Intent 
Feedback tickets: 
[GitHub #58660 - Design Feedback](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/gh/department-of-veterans-affairs/va.gov-team/58660)
[GitHub #58714 - IA Feedback](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/gh/department-of-veterans-affairs/va.gov-team/58714)
[GitHub #58782 -Accessibility Feedback](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/gh/department-of-veterans-affairs/va.gov-team/58782)
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [x] Already completed, no assistance needed.
- [ ] Unknown at this time.
##### Midpoint
- [x] Accessibility
- [x] Content
- [x] IA
- [ ] Translation
- [ ] Already completed, no assistance needed.
- [ ] Unknown at this time.
##### Staging
- [x] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed.
- [x] Unknown at this time.
  
## Supporting artifacts
*Please provide supporting artifacts as available.*

- [Link to product outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/burials-memorials/pre-need/initiative-brief.md) 
- [Link to or attach user flows](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/burials-memorials/pre-need/research/pre-integration-research-may23/Pre-need%20_Pre-Integration_Process-Diagram_052223.png)
- [Link to prototype or mockups - Usability testing prototype](https://bahdigital.invisionapp.com/console/share/ET2UVYD7NKM)
- [Link to your research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/burials-memorials/pre-need/research/pre-integration-research-may23/pre-integration-research-plan.MD) 
- Link to any epics/issues that may be helpful: https://github.com/department-of-veterans-affairs/va.gov-team/issues/58437

#### Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [x] No, let's work asynchronously and meet if needed

 **Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.
 
### Accessibility Help in Slack
The [#accessibility-help channel](https://dsva.slack.com/archives/C8E985R32) in the DSVA Slack, is also available to you as a resource. Everyone is welcome to ask questions or get help from our a11y (accessibility) specialists. Tag `@Terry Nichols` to get a11y help asap.

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag `@Terry Nichols`.**

- If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
- If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)
 
 - If you are creating an [experimental design,](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) please [also contact the DST. ](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- If you are requesting [a11y support for user research,](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name) please complete this form, next. 

```[tasklist]
### Tasks
```

**Comment by coforma-terry** [comment]
Hey @ccruzgra - just letting you know we haven't forgotten about this and it is on our radar!
CC @laurwill

**Comment by ccruzgra** [comment]
@laurwill We were advised by our client today to remove the "under 18" copy from the static page as it isn't accurate. I noted it in the above AC as this:

> There is guidance under the By Mail copy for How do I apply? section that mentions "If that person is under 18 years" guidance that the client wishes to have removed as children under 18 with eligible Veteran sponsors, their eligibility would be determined at time of need and pre-need is not intended for them.

**Comment by ccruzgra** [comment]
@laurwill During our midpoint review, they suggested updating the green apply button to an action link on the pre-need static page. Is that something your team would take care of or would someone else need to make that update?

**Comment by laurwill** [comment]
Hi @ccruzgra , We can make that change from button to action link as part of our updates to the static page.

**Comment by ccruzgra** [comment]
@laurwill One thing that we noticed is that there is confunsing copy under "How do I prepare?" subway step 3 for gathering documents. It states to "Please see the list below" but the list is much further down, especially on mobile, and it might make sense to word this differently or think about how users are being directed to see the lists for what info they need to apply.

**Comment by aliyahblackmore** [comment]
Thanks for this flag @ccruzgra. We'll plan to assess and adjust the content under "How do I prepare" as part of our updates to the static page. 

cc: @laurwill

**Comment by ccruzgra** [comment]
@laurwill As part of our [accessibility feedback](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63190) for our Midpoint Review, @briandeconinck suggested adding "(opens in a new tab)" to the end of external links. We are doing this to the external links in our form, but is this something your team could do for the external links on the two static pages in this issue?

**Comment by aliyahblackmore** [comment]
Hi @ccruzgra - thanks for raising this midpoint review feedback! 

I just took a look at the two static pages linked above in this ticket and right now there aren't any links that are opening in a new tab. 

https://staging.va.gov/burials-memorials/pre-need-eligibility/
https://staging.va.gov/burials-memorials/pre-need-eligibility/after-you-apply/

Would you all like for the "Find a VA cemetery" link to open in a new tab, similar to the placement of this link on the Burial benefits page in the form?

We'll assess and make any needed adjustments to content on any links to address the midpoint feedback and align it with VA site style when we're making updates to these pages.

Thanks for your continued collaboration on this work.

cc: @laurwill

**Comment by ccruzgra** [comment]
Hi @aliyahblackmore. Yes, it makes sense to do that for the "Find a VA cemetery" link and any other link you think would benefit from opening in a new tab. Thank you for your help

**Comment by carly-yamrus** [comment]
@aliyahblackmore  @laurwill Here are the compiled suggestions for the Pre-Need information/static page and "after you apply" page. 

**Suggestions for VA.gov Pre-Need Information/static page:** 

- Descriptive use cases that are informative for all situations and scenarios 

> Example: Adult disabled child 

- Content update to the eligibility description (currently has negative verbiage) 

> “A Veteran who didn’t receive a dishonorable discharge.” 
> “I was a little confused about the first part find out if you are eligible.” “Why does is start out with what is not instead of saying what is. I would like to see the positive before the negative.” 

- Reduce content length for easier process to apply.  

> Participants had difficulty navigating this page on mobile devices. “Apply Now” button was lost between text. Some users struggled to find apply button. 

- Condense the information and rearrange based on priority.  

> Participants were overwhelmed with repetitive information.“What information do I need to apply? It’s like, we’ve already gone through that at the top. It can get overwhelming. It’s already a sensitive subject, burying your loved one.” “I get it you want to give the information up front but it’s a sensitive subject it's just overwhelming. It could be less wordy.” 

To consider: 

> An additional “Apply Now” button near the top of the page or higher up on the page 


**Suggestions for VA.gov Pre-Need After You Apply page:** 

- Provide an estimate/timeline of events to expect after the application is submitted as some participants found the language to be too vague. 

- Address tracking after application submission. 

> Note: Claims Status Tool does not have the ability to track Pre-Need submissions 

- Provide additional context around benefits and helpful details surrounding burial. 

> Users were curious about additional burial-related benefits that they or their loved ones may be eligible for, such as survivor benefits. “A lot don’t know about survivor benefits”. 

> Some people may not know about the monetary benefits the government offers in these situations 

> One user remarked that she didn’t know what a burial liner is.

**Comment by ccruzgra** [comment]
@laurwill @aliyahblackmore Just to add on to Carly's post these are the suggestions from the Integration study participants regarding those two static pages.

**Comment by aliyahblackmore** [comment]
Hi @carly-yamrus and @ccruzgra thanks so much for passing along these compiled suggestions!

We'll plan to start this work again in our upcoming sprint.

I will take a close look at the above questions and include any additional questions in this ticket.

Would it be possible for you all to share a link to the full research readout?

**Comment by ccruzgra** [comment]
@aliyahblackmore No problem! Let us know if you need further context for any of the above.

**Comment by ccruzgra** [comment]
Hi @aliyahblackmore It's similar to what we discussed for the actual form. Parents of service members could apply for pre-need and be deemed eligible under specific circumstances based on the Corey Shea Act at the time of their service member's passing. So pre-need is open to them.

**Comment by aliyahblackmore** [comment]
Yes, appreciate you clarifying! I wanted to confirm as the PDF linked above does have a point about eligibility determination being at time of need and doesn't mention pre-need. I'll note that pre-need is open for parents as well. 

Thank you!

**Comment by aliyahblackmore** [comment]
Hi @ccruzgra  yes, appreciate you clarifying! This was still an open question that you mentioned you'd follow up about on the form work, so I wanted circle back to confirm as the PDF linked above does have a point about eligibility determination being at time of need and doesn't mention pre-need. I'll note your confirmation that pre-need is open for parents as well.

Thanks so much!

**Comment by aliyahblackmore** [comment]
Hi @ccruzgra yes! When I was looking back at the questions and replies in the form intake request, I noticed that the piece about time of need vs. pre need was still an open question that you mentioned you'd follow up about on the form work. So I wanted circle back just to confirm as the PDF linked above does have a point about eligibility determination being at time of need and doesn't mention pre-need. I'll note your confirmation that pre-need is open for parents as well.

Thanks so much!

**Comment by aliyahblackmore** [comment]
Hi @ccruzgra on the [After you apply static page](https://staging.va.gov/burials-memorials/pre-need-eligibility/after-you-apply/), [Carly suggested](https://github.com/department-of-veterans-affairs/va.gov-team/issues/61517#issuecomment-1698013066) providing an "estimate/timeline of events to expect after the application is submitted as some participants found the language to be too vague."

There isn't any more information about a time estimate included here, but I'd like to address this feedback in the content. Do you all have an estimate/average time for when applicant's would expect to recieve a decision about their pre-need application? 

As an example, [this health care application after you apply page](https://www.va.gov/health-care/after-you-apply/) includes the time called out.

Thank you!

**Comment by barron-charles** [comment]
Hi @aliyahblackmore,

I have a few more findings and recommendations we found for the information page:
1. Make sure to use 'supporting files' instead of 'supporting documents' terminology.
2. The current page has United States Soldiers and Airmen’s Home National Cemetery. The correct text is United States Soldiers' and Airmen's Home National Cemetery.
3. "To apply, you’ll need your (or your sponsor’s):", isn't bolded under, What information do I need to apply?
4. The subway map could have reduced left padding to allow for less vertical scrolling. See the subway map on the introduction page to compare for this one.

**Comment by aliyahblackmore** [comment]
Hi @barron-charles - thanks for sharing these additional recommendations. 

I'll assess and make adjustments to the content.

**Comment by barron-charles** [comment]
@aliyahblackmore For the last two sections, 'What if I need help filling out my application?' and 'What happens after I apply?', the contact information should match most recent updates to the form. You can find the formatting on the bottom of every page in the pre-need integration mockups to reference.

New changes:
"Call the National Cemetery Scheduling Office at [800-535-1117](tel:8005351117) (TTY: 711), and select option 4. We’re here Monday through Friday, 8:00 a.m. to 5:30 p.m ET."

Information page currently:
1.  'What if I need help filling out my application?'
- Call our toll-free hotline at [800-535-1117](tel:8005351117) ([TTY: 711](tel:711)). We’re here Monday through Friday, 8:00 a.m. to 7:30 p.m. ET, and Saturday, 9:00 a.m. to 5:30 p.m. ET.

2. What happens after I apply? 
- For more information, you can call [800-535-1117](tel:8005351117) ([TTY: 711](tel:711)). We’re here Monday through Friday, 8:00 a.m. to 7:30 p.m. ET, and Saturday, 9:00 a.m. to 5:30 p.m. ET.

**Comment by ccruzgra** [comment]
Hi @aliyahblackmore let me know if this helps

> Question 1 - On the [After you apply static page](https://staging.va.gov/burials-memorials/pre-need-eligibility/after-you-apply/), https://github.com/department-of-veterans-affairs/va.gov-team/issues/61517#issuecomment-1698013066 providing an "estimate/timeline of events to expect after the application is submitted as some participants found the language to be too vague."
> 
> There isn't any more information about a time estimate included here, but I'd like to address this feedback in the content. Do you all have an estimate/average time for when applicant's would expect to recieve a decision about their pre-need application?
> 
> As an example, [this health care application after you apply page](https://www.va.gov/health-care/after-you-apply/) includes the time called out.

We were advised by our client to hold off on updates regarding decision times until after Integration changes have been stabilized, so it's something we'll need to keep in mind as a future enhancement.

> Question 2- And there's currently a telephone number under "How long does it take VA to make a decision?" Is there a certain amount of time that an applicant should wait before they call?

We have asked before and the answer we have received is it depends and NCSO/EO office does not want to put a timeline on it to the public, at least for now.

**Comment by ccruzgra** [comment]
Hi @aliyahblackmore @laurwill Is there an estimate on when we'll be able to take a look at your planned changes and recommendations for the Pre-Need static pages?

**Comment by aliyahblackmore** [comment]
Hi @ccruzgra - Laura finished her review of my updates earlier today and I'll get the changes in Drupal (the CMS) this week. 

After that, we'll need to send these pages over to the SMEs to have them confirm any last questions and then review and approve the updated content. We're happy to send over the email and will be sure to include you on the thread!


cc: @laurwill

**Comment by ccruzgra** [comment]
Thanks, @aliyahblackmore and @laurwill!

**Comment by aliyahblackmore** [comment]
Hi @ccruzgra - we've made updates to the static pages based on the previous stakeholder requests and research feedback that you all posted above.

Before we send over an email to the SMEs, do you need to review the static page updates? If so, we can share the pages with you here.

Also flagging that I'll be OOO starting tomorrow (11/21) until Friday (11/25). If there are any urgent questions that come up tomorrow or early on Wednesday, please reach out to Laura. And also, please note that many folks on the content and larger CAIA team will also be out later in the week, so someone will reply to any urgent questions when they can.

**Comment by ccruzgra** [comment]
Yes, we can review the updates before you send them out, @aliyahblackmore. We have a story for that planned this sprint, so that would be perfect. Happy Holidays!

**Comment by aliyahblackmore** [comment]
Hi @ccruzgra! 

Here are the links to the static pages for you to review ahead of us sharing them with SMEs. 

Also noting that we made updates to the Eligibility for burial in a VA national cemetery page to reflect that parents of service members are also eligible for burial benefits. And we included more details about parent eligibility under the Corey Shea Act as well as information about who VA considers an adult dependent child. This page is now linked in what will be a featured content box on the How to apply page.

[Eligibility for burial in a VA national cemetery](https://preview-prod.vfs.va.gov/preview?nodeId=856)

[Pre-need eligibility for burial in a VA cemetery](https://preview-prod.vfs.va.gov/preview?nodeId=862)

[After you apply](https://preview-prod.vfs.va.gov/preview?nodeId=863)

Notes on some of these changes:

**Eligibility and How to apply pages**

• We made adjustments to the Eligibility page to reflect that parents of service members are also eligible for burial benefits. And we included more details about parent eligibility under the Corey Shea Act as well as information about who VA considers an adult dependent child. This page is now linked in what will be a featured content box (a blue box) at the top of the How to apply page.
• On the How to apply page, we made some content adjustments to align with changes in the form intro page/form content.
•  We also removed the process list on the page to align with the style of more recent Eligibility pages on VA.gov.

**After you apply page**

• We removed information about NCA processing applications, as requested by stakeholders.
• We updated links for information about the different review and appeal options and also made some adjustments to the reviews/appeals language.

Please tag Laura if you all have any questions that come up tomorrow or early on Wednesday. 


cc: @laurwill

**Comment by ccruzgra** [comment]
@aliyahblackmore @laurwill Really beautiful job on the content changes to all three pages! Everything is so much more concise without the user missing out on what they need to see, so fantastic job with all of the copy and design updates.

We have a few suggestions for you:

**Eligibility for burial in a VA national cemetery page**
- Eligibility information for certain family members section
    - Adult dependent children: Language around “enrolled as a full-time student”. We had been advised that they keep the current language somewhat vague to allow for flexibility in this eligibility requirement. Rhonda Jones (BTRS) suggested it’s vague for a reason, may have a broad definition of full-time course of instruction, and don’t want to limit them to full-time college or vocational school.

**How to apply page**
- Featured content box
    - “Soldiers’” missing apostrophe per the name of the cemetery: https://www.arlingtoncemetery.mil/Explore/Soldiers-and-Airmens-Home-National-Cemetery 
- How to apply section
    - Option 2 for mail
        - “NCA Evidence Intake Center” is preferred for the address, not “NCA FP Evidence Intake Center” per stakeholder guidance.
        - By mail or fax, “Application” should be lowercase
- Information you’ll need to apply section
    - Instead of “need to” when referring to providing supporting documents, we recommend using language that makes it clear they don’t need to provide documents, but that they can provide them to help process application faster as it’s not required to provide anything to apply.
    - Instead of using “director” for guidance around institutions providing care, we opted to leave “principal officer” in our supporting files page guidance since the language is more generic.
    - If applying for an adult dependent child, shouldn’t the guidance under “applying on behalf of someone else” also apply? So, in addition to the disability documents, they could provide proof that they can apply on someone’s behalf.

**After you apply page**
No feedback   

Again, wonderful job with these changes, and we're really excited for them to go live once final approvals are in. We will share your changes with @CatherineHughes202 once she's back from leave on 11/27 in case she has additional feedback. Thank you!

cc: @kelby-stewart

**Comment by aliyahblackmore** [comment]
Hi @ccruzgra - thanks so much for this feedback.

Let us know if @CatherineHughes202 has any additional feedback and we'll make updates to the pages ahead of sending them out the NCA stakeholders. 

Thanks so much for your continued collaboration!

**Comment by ccruzgra** [comment]
@aliyahblackmore an update for you. We'll be taking a look at the content updates with Cat Hughes tomorrow and will advise you if there any additional recommendations from our end.

**Comment by aliyahblackmore** [comment]
Hi @ccruzgra just an update from me!

I needed to work on a few other priorities this week, but I'm almost done reviewing the feedback from you and Cat. I'll continue on this early next week, pass feedback to Laura, and let you all know if we have any other questions ahead of sharing the pages with SMEs. 

Here's one question that I have so far:

**The the feedback/notes on the "need to" language -**  the bullet points in that list are required fields on the forms, yes? The DD214 is optional and we use language ("you can") to communicate that providing it will help us process the application faster. Do you all have concerns about the DD214 language and whether there will be confusion about that being required? If so, we can address that in the content.

Thanks for your continued collaboration and more from me soon.

**Comment by aliyahblackmore** [comment]
Hi @ccruzgra!

I just want to clarify what's required when someone submits their application so that we can provide a few more details on the static page.

From our previous review of the form flow via the content source of truth link, our adjustments frame the documents in the accordions **"If you're applying on behalf of someone else" and "If you're applying for the adult dependent child of a Veteran"** as additional documents required when submitting the application. For example, if someone is applying on behalf of someone else they need documents that prove that they can in fact apply on behalf of that person.

But from your above comment, it sounds like these documents are helpful and recommended when applying online, but not required when they submit their application online. 

**If someone decides not to upload these documents with their form, will they eventually need to send these documents later on and will VA ask them to send them later on?**  

We'll want to adjust the content on the static page and in the form to address this so that's it's clear what the person applying may need to do later on if they choose not to upload documents while applying online.

**Comment by ccruzgra** [comment]
Hi @aliyahblackmore! Just an update on the above question. NCSO agents can request things like service records, but if they need additional information from applicants, they'd likely be reaching out to gather that. We'll try to get more information from you next week since we have quite a few people out this week on our end, too.

**Comment by aliyahblackmore** [comment]
HI @ccruzgra 

I got back from my OOO time yesterday so just getting to this ticket comment.

Yes, if you can confirm whether they'll reach out for those additional documents after submission or whether the person applying should automatically send those documents after submitting the application, that would be great!  And we can adjust this on the static page and on the form.

**Comment by ccruzgra** [comment]
@aliyahblackmore the answer we heard is that there are too many scenarios so it's hard to get specific with laying out  requirements. We should encourage them (not require them) to submit documents but advise that, if additional documents are needed, eligibility agents could reach out to applicants or preparers for that information. I hope that helps!

**Comment by aliyahblackmore** [comment]
Thanks so much @cindymerrill! 

We have some suggested language in our form feedback and in the static page updates to address this (_"If you don't have these documents, you can submit them after you apply"_), so when we share the pages with SMEs we'll be sure to have them confirm that language is okay.

**Static page updates:** 

On Friday I added replies to the static page feedback[ in the mural you shared](https://app.mural.co/t/bahdigitalexperience6902/m/bahdigitalexperience6902/1701192365402/b02f08c213015e59c4ab191b14f491185e7ce9ea?sender=b19de862-306d-4228-8154-4ab64e3adcf4) with me above. There's just one question on there for the product team (in pink) and remaining questions are labeled for SMEs.

I'll have the content updates in Drupal tomorrow and then Laura and I will plan email to email SMEs with the updates and remaining questions. As I mentioned on Slack a few weeks ago, when we make any updates to our static pages on VA.gov the content team sends the emails to SMEs. In our emails, we usually call out specific changes made (i.e. outlining the specific text with a box) for their accuracy review and approval and list any remaining questions we have. 

Are there other ways you all have called out content updates to SMEs that may be helpful here? We can definitely take that in to consideration.

Thanks so much for your continued collaboration, Cindy.

**Comment by cindymerrill** [comment]
I'm not the Cindy you wanted to reply to. Perhaps @ccruzgra is?

**Comment by aliyahblackmore** [comment]
Apologies @cindymerrill! I'm going to untag you and tag the correct Cindy. Thanks for flagging!

**Comment by ccruzgra** [comment]
Hi @aliyahblackmore. Thanks for so much for sharing that. We'll take a look at the comments you made in Mural. We had a similar idea in mind if we needed to call out the content changes to SMEs so I'm glad that's part of your process. We were thinking of outlining the changes in Mural with the before and after like we did in the Mural so it's easy to compare for SMEs. Having a visual is definitely helpful!

**Comment by aliyahblackmore** [comment]
Thanks @ccruzgra! I'll be getting the confirmed updates in Drupal - I'll ping you when they're ready!

**And in reply to your above response to our question:** 
 
Is it that case foor the PDF form that someone submits via mail/fax **and** the online form that the applicant has to sign the form, unless they're not able to do so? Or is signing the form only a requirement for the PDF version of the form (i.e. if they're filling out the PDF form and mailing or faxing it in)?

We ask this because the version of the online form content that we reviewed doesn't currently have specific instructions about signing the form and we just want to clarify if this requirement is the same for both those filling out and mailing/faxing PDF form and for those submitting the online form. 

If so, we'd consider adjustments to the static page and the online form content to clarity that this is a requirement for this form - whether online, mail, or fax.

And if this isn't the case, we'd need to adjust content on the static page to specify the signature requirements for forms submitted via mail or fax.

**Comment by ccruzgra** [comment]
@CatherineHughes202 Do you have feedback on requirements for Aliyah's question above?

**Comment by aliyahblackmore** [comment]
Thanks @ccruzgra ! 

I'll wait for a reply from Cat on that question. And if needed, we can flag that as a question for SMEs.

And there's one other product question [I added](https://app.mural.co/t/bahdigitalexperience6902/m/bahdigitalexperience6902/1701192365402/b02f08c213015e59c4ab191b14f491185e7ce9ea?wid=0-1704838851119) while I was making the Drupal updates. I'm wondering if the phone number selection option (4) and hours are the same if someone is requesting a burial?

**Comment by CatherineHughes202** [comment]
@aliyahblackmore cc @ccruzgra this is a requirement across the board and the guidance is applicable for mail, fax or online respectively

**Comment by aliyahblackmore** [comment]
Hi @barron-charles ! Cindy let us know that Laura and I should reach out to you with any questions.

**As we're wrapping up our content adjustments on the static pages before we share them with SMes, Laura has one other question:** 

Are there other documents that someone needs to submit showing that they can "submit" this online application on behalf of someone else? The static page is currently missing content that specifies "I can submit for this person." And the online form content is missing content that confirms that the person they're applying for "can't do it (i.e. submit) themselves."

cc: @laurwill

**Comment by carly-yamrus** [comment]
Hi @aliyahblackmore and @laurwill ! We have begun our initial discovery of the Headstones and Markers memorial items form this week and would like to make a recommendation for the Headstones, Markers, Medallions "Eligibility" static page. The language at present suggests this form is for living Veterans and active duty service members - as the title states "For Veterans and active-duty service members to get a headstone or marker." We are recommending adding the "deceased" language here to clarify. Thank you!

**Comment by aliyahblackmore** [comment]
Hi @carly-yamrus ! Thanks for looping us in to this work! 

Because the headstones, markers, and medallions form and static page work is separate from the pre-need eligibility determination form and static page work, I think we'd need you all to open up a separate ticket to track these updates.


Looping in our PM @strelichl here. Is that correct?

**Comment by barron-charles** [comment]
Hey @aliyahblackmore in terms of _can_, are you referring to the person's stability and ability filling it out? We have required forms Appointment of Veterans Service Organization as Claimant’s Representative (VA Form 21-22) and Appointment of Individual as Claimant’s Representative (VA Form 21-22a) when applying for someone else.

In terms of showing content that the person they're applying for "can't do it, (i.e. submit) themselves, @CatherineHughes202 any further thoughts with this one?

**Comment by aliyahblackmore** [comment]
Hi @barron-charles and @CatherineHughes202

**Circling back here! @CatherineHughes202 do you have any further thoughts on Laura's question:** 

_Are there other documents that someone needs to submit showing that they can "submit" this online application on behalf of someone else? The static page is currently missing content that specifies "I can submit for this person." And the online form content is missing content that confirms that the person they're applying for "can't do it (i.e. submit) themselves."_

And of course, if needed, we can flag this question for SMEs when we share the updates with them.

**Comment by barron-charles** [comment]
Hey @aliyahblackmore, I looked a bit more into this one, and I believe it's It's related to this portion from https://www.va.gov/burials-memorials/pre-need-eligibility/. 

**If you’re applying for an unmarried adult child of a Veteran or service member**, you’ll also need to provide supporting documents with information about the child’s disability. The Veteran or guardian of the child should also ask the child’s current doctor to verify the documents. These documents should include all of the following information:
The date of the disability’s onset, **and**
A description of the disability, mental or physical, **and**
A description of how dependent the disabled child is on the Veteran, **and**
The marital status of the child

So basically, the documentation proves that the person they're applying for can't do it.

**Comment by laurwill** [comment]
@bethpottsVADEPO here's the static page ticket for pre-need eligibility that we talked about, I just reassigned to you. Thanks!

**Comment by mnorthuis** [comment]
@bethpottsVADEPO if this work is complete, feel free to close this issue.

**Comment by bethpottsVADEPO** [comment]
@mnorthuis Closing this ticket as work is complete.
